apis/controller/v1alpha1/sliceconfig_types.go

```diff
@@ -106,8 +106,26 @@ type SliceNamespaceSelection struct {
 	Clusters  []string `json:"clusters,omitempty"`
 }
 
+type KubesliceEvent struct {
```

Comment:
Do we actually need this? Without this as well, when we do `kubectl describe sliceconfig <name>`, we will get all the events associated to that slice config within the CR.

Reply:
Kubernetes events are transient in nature. we want to persist the events from some of the action in this use case. 
Moreover these events would be coming from API-GW which does not have any framework to implement to ensure consistency in event generation.

Reply:
Cool. That make sense.